web/packages/design/src/DataTable/Table.tsx

```diff
@@ -288,21 +290,25 @@ function PagedTable<T>({
 
   return (
     <>
-      <StyledPanel>
-        <InputSearch
-          searchValue={searchValue}
-          setSearchValue={setSearchValue}
-        />
-        {(showTopPager || showBothPager) && (
-          <ClientSidePager
-            nextPage={nextPage}
-            prevPage={prevPage}
-            data={data}
-            {...fetching}
-            {...pagination}
-          />
-        )}
-      </StyledPanel>
+      {(isSearchable || showTopPager || showBothPager) && (
```

Comment:
Something to consider: if you don't want to show `StyledPanel` when none of those conditions are true but you also don't want to repeat all conditions upfront, you can use the `:empty` selector to hide `StyledPanel` if it has no children.

I ran into a similar situation a couple of months ago:

https://github.com/gravitational/teleport/blob/9383c38bf0f9fbc633269433398e8b68bba2b034/web/packages/teleterm/src/ui/Vnet/VnetSliderStep.tsx#L55-L92